src/main/java/de/lh/tool/service/entity/impl/ItemNoteServiceImpl.java

```diff
@@ -29,39 +37,86 @@
 	private ItemService itemService;
 	@Autowired
 	private UserService userService;
+	@Autowired
+	private UserRoleService userRoleService;
 
 	@Override
 	@Transactional
 	public Collection<ItemNoteDto> getDtosByItemId(Long itemId) throws DefaultException {
-		return convertToDtoList(itemService.findById(itemId)
-				.orElseThrow(() -> new DefaultException(ExceptionEnum.EX_INVALID_ID)).getItemNotes());
+		Item item = itemService.findById(itemId).orElseThrow(ExceptionEnum.EX_INVALID_ITEM_ID::createDefaultException);
+		if (!itemService.isViewAllowed(item)) {
+			throw ExceptionEnum.EX_FORBIDDEN.createDefaultException();
+		}
+		List<ItemNote> notes = item.getItemNotes().stream()
+				.sorted(Comparator.comparing(ItemNote::getTimestamp).reversed()).collect(Collectors.toList());
```

Comment:
Sinnvoll direkt Comparable in ItemNote implementieren?
(um einheitliches Verhalten sicherzustellen)